src/xtopcom/xvledger/src/xvaccount.cpp

```diff
@@ -130,6 +130,182 @@ namespace top
                 return {};
             }
         }
+
+        bool xvaccount_t::check_address(const std::string & account_addr, bool isTransaction)
+        {
+            // check address min-length and max-length
+            if(account_addr.size() <= enum_vaccount_address_prefix_size || account_addr.size() > enum_vaccount_address_max_size) {
```

Comment:
split this 200 lines single function into several parts?

Reply:
next version fix it